wordpress/wp-content/mu-plugins/cds-base/classes/Modules/Cleanup/Roles.php

```diff
@@ -8,8 +8,7 @@ class Roles
 {
     public function __construct()
     {
-        Utils::checkOptionCallback('cds_base_activated', '1.0.11', function () {
-
+        Utils::checkOptionCallback('cds_base_activated', '1.0.15', function () {
```

Comment:
Could be `1.0.12`, but it doesn't really matter.

Reply:
Will leave as is --- I was testing locally a bunch of times.